dbt_subprojects/hourly_spellbook/models/_project/cow_protocol/arbitrum/cow_protocol_arbitrum_eth_flow_orders.sql

```diff
@@ -0,0 +1,62 @@
+{{  config(
+        alias='eth_flow_orders',
+
+        materialized='incremental',
+        partition_by = ['block_date'],
```

Comment:
over time, we have found that we need larger partitions. can you introduce `block_month` in query below, similar to `block_date`, then update here to monthly level?
you can either override `block_date` or add in addition to it, depending on if you use the column in queries downstream